unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerInfo/PassportPlayerInfoComponentView.cs

```diff
@@ -206,7 +237,40 @@ private void CopyWalletToClipboard()
             if(fullWalletAddress == null)
                 return;
 
-            GUIUtility.systemCopyBuffer = fullWalletAddress;
+            OnWalletCopy?.Invoke();
+            Environment.i.platform.clipboard.WriteText(fullWalletAddress);
+            if (copyAddressRoutine != null)
+            {
+                StopCoroutine(copyAddressRoutine);
+            }
+
+            copyAddressRoutine = StartCoroutine(ShowCopyToast(copyAddressToast));
+        }
+
+        private void CopyUsernameToClipboard()
+        {
+            if(string.IsNullOrEmpty(model.name))
+                return;
+
+            Environment.i.platform.clipboard.WriteText(model.name.Split('#')[0]);
+            if (copyNameRoutine != null)
+            {
+                StopCoroutine(copyNameRoutine);
+            }
```

Comment:
Only a little detail: I'm seeing (not only in this class but in the rest of them) that you're using the `{`...`}` in some conditions with only 1 line in its content, and you're not using them in another ones. This sometimes creates a bit of "visual noise" in the code, so it would be nice to choose a way and apply it for all the cases, at least in a same class.

Reply:
I tend to avoid using the {} when there is a return statement just because generally we don't add other code there, I use it in other cases because it's easier if we need to add new lines, I cannot remember, what is the standard we agreed on? So I can convert everything to the standard :)